runtime/integration-tests/src/liquidity_pools/pallet/development/tests/liquidity_pools/foreign_investments.rs

```diff
@@ -103,7 +103,7 @@ mod same_currencies {
 		Development::execute_with(|| {
 			setup_pre_requirements();
 			let pool_id = DEFAULT_POOL_ID;
-			let amount = 100_000_000;
+			let amount = 10 * dollar(12);
```

Comment:
The issue we were facing was that we minted less than the ED, so `mint_into` threw.

Reply:
Ouch!! Good catch, I thought that you could mint more that ED if the target account has more than `ED`